src/Configuration.php

```diff
@@ -134,8 +153,36 @@ public function getDOMDocument()
      * @param  string     $xpathExpression
      * @param  boolean    $alwaysReturnArray
      * @return array|null
+     * */
+    public function get($xpathExpression, $alwaysReturnArray = false) {
+        if ($this->useCaching && ($alwaysReturnArray === true) ) {
```

Comment:
Please use strict checking for the boolean. The parentheses can be removed. Also there's a space between the asterisks.